swift3/controllers/multi_upload.py

```diff
@@ -602,6 +602,10 @@ def POST(self, req):
         container = req.container_name + MULTIUPLOAD_SUFFIX
         resp = req.get_response(self.app, 'GET', container, '', query=query)
         objinfo = json.loads(resp.body)
+
+        # pylint: disable-msg=E1103
+        objinfo.sort(key=lambda o: int(o['name'].split('/')[-1]))
```

Comment:
Shouldn't you use int(X,16) since it's a hexadecimal?

Reply:
No, the partNumber is only decimal, you may be confused with UploadID which is hexadecimal